app/models/user.rb

```diff
@@ -0,0 +1,33 @@
+require 'data_mapper'
+require 'bcrypt'
+
+class User
+  include DataMapper::Resource
+
+  property :id,              Serial
+  property :name,            String
+  property :username,        String, :unique => true
+  property :email,           String, :unique => true
```

Comment:
Style/HashSyntax: Use the new Ruby 1.9 hash syntax. (https://github.com/bbatsov/ruby-style-guide#hash-literals)